src/OpenLoco/Paint/Paint.h

```diff
@@ -200,8 +200,8 @@ namespace OpenLoco::Paint
         /*
          * @param rotation @<ebp>
          * @param imageId  @<ebx>
-         * @param offset_x @<al>
-         * @param offset_y @<cl>
+         * @param offsetX @<al>
+         * @param offsetY @<cl>
          * @param offset_z @<dx>
```

Comment:
offsetZ

Reply:
should we delete these doxygen comments? doesn't seem like they're used or updated